### PR TITLE
Use standard Enketo function to ignore irrelevant

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

Instead of having our own code to strip out irrelevant fields we
can now use pass `irrelevant = false` as an option to Enketo.

medic/medic-webapp#3449

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: static/js/services/enketo-translation.js [hunk 1]
```diff
@@ -352,8 +343,7 @@ angular.module('inboxServices').service('EnketoTranslation', [
       }
 
       withElements(root.childNodes)
-        // TODO repeat-relevant may be unnecessary if https://github.com/enketo/enketo-core/issues/336 is resolved
-        .filter(without('meta', 'inputs', 'repeat', 'repeat-relevant'))
```
Comment (alxndrsn) [inline static/js/services/enketo-translation.js]: Do we want to deprecate this, but continue filtering `repeat-relevant` nodes until we've removed them from all the production forms?

Comment (garethbowen) [reply static/js/services/enketo-translation.js]: We could do, but in my tests it made no difference - the field didn't exist in the created contact or its children. How about I put a line in the release notes about removing the `repeat-relevant` nodes?

Comment (alxndrsn) [reply static/js/services/enketo-translation.js]: I'll add a warning to `medic-conf` flagging it as unnecessary with webapp `2.14.0` and later.

Comment (garethbowen) [reply static/js/services/enketo-translation.js]: Nice one.

**Pull request merged**